### PR TITLE
Add method to visualize plots by figures

### PR DESCRIPTION
**Pull request by zhourunlai**
Add method to visualize accuracies and loss vs epoch for training and validation data

Parameter:
1. history (Necessary, an instance returned by model.fit method)
2. figure_name (Optional, a name to plots, default set to "plots")

Usage:

```
from keras.utils.visualize_util import figures
hist = model.fit(X,y)
figures(hist)
```

### CODE REVIEW

**Comment by fchollet** [comment]
This example makes a lot of assumption (that there is validation data and that the only two metrics are accuracy and loss). I would agree with having utilities to generate nice plots out of a `History` objects, but it would need to be general, rather than having a specific use case hard-coded into it.
